packages/manager/modules/pci/src/projects/project/storages/databases/add/command/command.component.js

```diff
@@ -0,0 +1,14 @@
+import controller from './command.controller';
+import template from './command.html';
+
+const component = {
+  bindings: {
+    goBack: '<',
+    projectId: '<',
+    data: '<',
+  },
+  template,
+  controller,
```

Comment:
`transclude: true` not required ?

Reply:
I have checked the other modal components we have in the project, and I have not found any with `transclude: true`.
Is there any benefit to have it in this case ?